app/models/publishable.rb

```diff
@@ -72,10 +98,12 @@ def publish_to_portals(self_url)
   def self.included(clazz)
     clazz.class_eval do
       after_update :auto_publish_to_portal
+      has_many :portal_publications, :as => :publishable, :order => :updated_at
     end
   end
 
   def auto_publish_to_portal
+    hash =
```

Comment:
this looks like an error, ruby probably doesn't complain because it sets the value of hash to the return value of logger